ProjectSettings/EditorBuildSettings.asset

```diff
@@ -8,15 +8,18 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/Scenes/MainGame/Boot.unity
     guid: 2df4d7ceea51a4f4c8597c7d9102de90
-  - enabled: 1
+  - enabled: 0
```

Comment:
You don't read my beautiful description do you :(, anyway to prevent merge conflicts on Ryans branch I put the input seed in a different scene and to test it I removed the Main Menu scene. Like mentioned above I will remove this before merging to develop

Reply:
Cool looks good then. Please remove it :)